src/stdatamodels/jwst/datamodels/wcs_ref_models.py

```diff
@@ -131,6 +155,34 @@ def __init__(
         output_units=None,
         **kwargs,
     ):
+        """
+        Initialize the model.
+
+        Parameters
+        ----------
+        init : str, None, optional
+            Input file name in ASDF format from which to initialize the model.
```

Comment:
This is in a few places, but I think the phrasing is a little awkward - the file is in ASDF format, not the file name.  I'd suggest instead: 
```suggestion
            File name for input file in ASDF format from which to initialize the model.
```

Reply:
Thankfully this was copy-pasted so it's easy to "replace all"